### ISSUE TITLE
Is the cache test allowed to load all at once?

### ISSUE THREAD

**Issue by atavistock** [TechEmpower/FrameworkBenchmarks#9308]
https://github.com/TechEmpower/FrameworkBenchmarks/blob/85ff4916681461ffe36724a91895cbc978fcf20b/frameworks/Ruby/rails/app/controllers/hello_world_controller.rb#L22

I was under the impression that this should be a loop doing multiple lookups that may or may not hit the cache.  This implementation is doing one large cache lookup and filling in the blanks.  Should I change other implementations to match?

**Comment by volyrique** [comment]
I don't think that there is any requirement like that. In fact [requirement 6](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#requirements-6) says the opposite:
> Additionally, if a separate caching process is used (see below), it is acceptable to fetch all of the randomly-selected objects using a single cache query operation.

**Comment by NateBrady23** [comment]
Closing as part of bulk cleanup.
